engine/class_modules/sc_rogue.cpp

```diff
@@ -4102,9 +4102,15 @@ struct vendetta_t : public rogue_spell_t
 
 struct stealth_t : public rogue_spell_t
 {
+  double precombat_seconds;
```

Comment:
Without evaluating if it's needed in the first place or not, times in simc are generally stored as `timespan_t`s.

Reply:
It's cast to `timespan_t` after the conditional, similar to how precombat is implemented on the other spells in this file.

Reply:
Yes, I saw that. I'm not going to press it (not "my" module etc) but I'd argue the other spells are doing it "wrong" too.

Reply:
I'd agree with that sentiment. Time is intended to be represented by `timespan_t` to avoid ambiguity between units.